scripts/setup-local.sh

```
@@ -243,10 +243,14 @@ configure_site_agent() {
         --tls-ca-path /var/secrets/temporal/certs/server-interservice/ca.crt \
         --tls-server-name interservice.server.temporal.local || true
 
+
+    local nico_address="${NICO_ADDRESS:-${CARBIDE_ADDRESS:-nico-rest-mock-core:11079}}"
+
     kubectl -n $NAMESPACE get configmap nico-rest-site-agent-config -o yaml | \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)